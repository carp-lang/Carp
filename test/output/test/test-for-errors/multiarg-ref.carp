multiarg-ref.carp:2:1 `ref` takes a single argument, but I got `(ref 1 2)`.
