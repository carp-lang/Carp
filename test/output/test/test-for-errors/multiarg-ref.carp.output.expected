multiarg-ref.carp:2:2 [ERROR] ref expected 1 arguments but received 2.

The arguments 2 are not needed.
